en/mulesoft/modules/ROOT/pages/ms_composer_flows.adoc

```diff
@@ -131,6 +132,22 @@ To repair a connection:
 . Click the change icon in the trigger or action that requires repair, and then click *Change Connection*.
 . Click *Repair*.
 
+== Delete a Connection
+
+If you no longer need a specific OAuth connection, you can delete it.
+
+When you attempt to delete a connection, Composer ensures that deleting the connection will not affect other flows.
+
+To delete a connection:
+
+. <<create-a-new-flow,Create a new flow.>>
+. In the *What Should Start This Flow?* window, click the change icon. +
+The current connections appear.
+. Next to the connection that you want to delete, click the trashcan. +
+A message appears ensuring that you want to delete the selected connection and lists any flows currently using that connection. If any flows are using that connection, you must first <<change-a-connection,change the connection>> for those flows.
```

Comment:
There is a bit more nuance here. I think it would be worthwhile to explain.

Scenario 1: The connection is not used in any flows
The user will be told that the connection is not used in any flows and they can proceed with deleting it
Scenario 2: The connection is used in inactive flows
The modal will show exactly how many (count) and which (names) inactive flows are using this connection. They will be warned that deleting this connection will affect these flows. 
When a user re-opens one of these flows later, they will see the existing error handling UI that tells them the connection is missing for a card and that they must select a new connection.
Scenario 3: The connection is used in active and inactive flows
The modal will show exactly how many active and inactive flows use this connection. It will also list the names of each of the flows and what their status is (active/inactive).
The modal should ask for secondary confirmation to confirm they really want to proceed.
If they chose to delete the connection, we will first attempt to stop the active flows and then will delete the connection.

@saikiran2012 - could you please confirm that the above behavior is all still accurate?

Reply:
yes, the behavior is accurate.